recipes/llvm-core/all/conanfile.py

```diff
@@ -102,8 +102,6 @@ def _configure_cmake(self):
         cmake.definitions['LLVM_ENABLE_WARNINGS'] = True
         cmake.definitions['LLVM_ENABLE_PEDANTIC'] = True
         cmake.definitions['LLVM_ENABLE_WERROR'] = False
-        cmake.definitions['LLVM_ENABLE_LIBCXX'] = \
-            'clang' in str(self.settings.compiler)
```

Comment:
These were the lines that were introducing the additional `stdlib` flag @uilianries (my fault 😅). Now, the only thing left is to find out why it does not find the standard library headers when compiling using `libc++`.